pkg/rhRegion/rhRegion.go

```diff
@@ -0,0 +1,50 @@
+package rhRegion
```

Comment:
so this whole file is a great example of what can get pushed down into the ocm-sdk for reuse in rosa cli. not necessarily now, but keep it in mind.

Reply:
Ack.